components/hab/src/cli.rs

```diff
@@ -893,8 +893,9 @@ pub fn sub_sup_run() -> App<'static, 'static> {
     // is displayed confusingly as `hab-sup`
     // see: https://github.com/kbknapp/clap-rs/blob/2724ec5399c500b12a1a24d356f4090f4816f5e2/src/app/mod.rs#L373-L394
     (usage: "hab sup run [FLAGS] [OPTIONS] [--] [PKG_IDENT_OR_ARTIFACT]")
-          (@arg LISTEN_GOSSIP: --("listen-gossip") env(GOSSIP_LISTEN_ADDRESS_ENVVAR) default_value(&GOSSIP_DEFAULT_ADDR) {valid_socket_addr}
+    (@arg LISTEN_GOSSIP: --("listen-gossip") env(GOSSIP_LISTEN_ADDRESS_ENVVAR) default_value(&GOSSIP_DEFAULT_ADDR) {valid_socket_addr}
         "The listen address for the Gossip System Gateway.")
+    (@arg LOCAL_MODE: --("local-mode") "Start the supervisor in local mode. This sets the gossip listen address to 127.0.0.2 effectively scoping it to only the local node. This is equivalent to using 127.0.0.2 as the ip for --listen-gossip. Note: Setting local mode will override values for --listen-gossip, including values set via the env variable.")
```

Comment:
This should probably be named something like `local-gossip-mode` since there are other things the supervisor listens for and this only applies to the gossip subsystem.